backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/internal/BpkAppSearchModalImpl.kt

```diff
@@ -36,6 +37,7 @@ internal fun BpkAppSearchModalImpl(
     results: BpkAppSearchModalResult,
     onInputChanged: (String) -> Unit,
     modifier: Modifier = Modifier,
+    clearAction: BpkClearAction? = null,
```

Comment:
clearAction should be required always for `BpkAppSearchModal`

Reply:
Have updated it.